### ISSUE TITLE
input box is hidden by keyboard

### ISSUE THREAD

**Issue by Stephen-PM** [cds-snc/covid-shield-mobile#646]
On iOS with large font the keyboard covers the input box. The user cannot see the number that they are typing in. The user cannot scroll to put the input box into view.

![image](https://user-images.githubusercontent.com/63243608/87254310-c385ae80-c44f-11ea-9ded-bccbbe504922.png)
